mantidproject/mantid
# Workspace dock deletion bugs

Fixes bugs with deleting workspaces from the Mantid dock.

**To test:**
Two things to test:
 - Try deleting a workspace using the delete key when the dock is not focused.
   - It should no longer be deleted
 - Try turning on delete confirmations (Properties -> General -> Confirmations) and check that
 you are asked to confirm before deleting a workspace.
 - Code review

Fixes #18886 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
